pinot-core/src/main/java/org/apache/pinot/core/common/Block.java

```diff
@@ -19,41 +19,7 @@
 package org.apache.pinot.core.common;
 
 /**
- *
- * A block represents a set of rows.A segment will contain one or more blocks
- * Currently, it assumes only one column per block. We might change this in
- * future
+ * A {@code Block} represents the data block returned by the {@link Operator}. Each operator can return multiple blocks.
```

Comment:
```suggestion
 * A {@code Block} represents the data block returned by the {@link Operator}. Each operator can return multiple blocks.
 * A {@code Block} can contain one or more columns of data depending on the operator implementation.
```

Reply:
It can contain document ids, column values (seems always from multiple columns) or result rows. Updated the javadoc